src/transformations/histogram.jl

```diff
@@ -39,34 +39,49 @@ Base.@kwdef struct HistogramAnalysis{D, B}
     bins::B=automatic
     closed::Symbol=:left
     normalization::Symbol=:none
+    visual::Union{typeof(automatic), Layer}=automatic
```

Comment:
Maybe it would be better to directly use `AlgebraOfGraphics.Visual` for this case? Maybe we need to export it then